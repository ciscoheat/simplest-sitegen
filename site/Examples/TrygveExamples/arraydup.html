<!-- build:title -->arraydup.k<!-- /build:title -->
<!-- build:content -->
<h3>trygve: arraydup.k</h3>
<pre><code class="language-java">class ArrayDupTest {
   public void test() {
      int [] intArray = new int[5];
      for (int i = 0; i &lt; 5; i++) {
         intArray[i] = i.clone()
      }
      for (int i = 0; i &lt; 5; i++) {
         System.out.println(intArray[i])
      }
      intArray = null;
      // System.out.println(intArray)
   }
}

{ ArrayDupTest adt = new ArrayDupTest() adt.test() }</code></pre>
<pre><code>/* GOLD:
___________________________________________________________
0
1
2
3
4
*/
</code></pre>
<p>&nbsp;</p>
<!-- /build:content -->
