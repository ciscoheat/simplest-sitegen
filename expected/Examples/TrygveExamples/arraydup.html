<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>arraydup.k</title>
  <link rel="stylesheet" href="/CSS/chota.min.css?glxzhr">
  <link rel="stylesheet" href="/CSS/fulloo.css?1dpkm1x">
  
</head>
  <body>
    
<h3>trygve: arraydup.k</h3>
<pre><code class="language-java">class ArrayDupTest {
   public void test() {
      int [] intArray = new int[5];
      for (int i = 0; i &lt; 5; i++) {
         intArray[i] = i.clone()
      }
      for (int i = 0; i &lt; 5; i++) {
         System.out.println(intArray[i])
      }
      intArray = null;
      // System.out.println(intArray)
   }
}

{ ArrayDupTest adt = new ArrayDupTest() adt.test() }</code></pre>
<pre><code>/* GOLD:
___________________________________________________________
0
1
2
3
4
*/
</code></pre>
<p>&nbsp;</p>

  </body>
</html>
